user-docs/modules/ROOT/pages/fedora-iot-bootc-quay-example.adoc

```diff
@@ -0,0 +1,128 @@
+= Fedora IoT Bootc Image Example with Quay
+
+== Building and Booting a Fedora IoT Bootc Image
+
+This example walks through building and booting a Fedora IoT bootc image using Quay.io, as well as pushing an update to a booted 
+Fedora IoT system.
+
+To start, clone the https://gitlab.com/fedora/bootc/base-images[Fedora Bootc Base Images] Git repository. This includes the Fedora
+IoT package manifest we will use to create our Fedora IoT bootc image.
+
+Within this clone, create a `Containerfile.custom` with the following contents:
+----
+FROM localhost/fedora-bootc as builder
+RUN /usr/libexec/bootc-base-imagectl build-rootfs --manifest=fedora-iot /target-rootfs
+
+FROM scratch
+COPY --from=builder /target-rootfs/ /
+LABEL containers.bootc 1
+ENV container=oci
+STOPSIGNAL SIGRTMIN+3
+CMD ["/sbin/init"]
+----
```

Comment:
See the comment on the `fedora-iot-bootc-pmachine-example.adoc` file about changes that should be made here as well